sphericart-jax/setup.py

```diff
@@ -9,7 +9,7 @@
 
 
 ROOT = os.path.realpath(os.path.dirname(__file__))
-SPHERICART_ARCH_NATIVE = os.environ.get("SPHERICART_ARCH_NATIVE", "ON")
+SPHERICART_ARCH_NATIVE = os.environ.get("SPHERICART_ARCH_NATIVE", "OFF")
```

Comment:
We might want to keep these ON for people who compile from source so they get optimal performance